vsphere/test_vsphere.py

```diff
@@ -335,3 +336,102 @@ def test_mor_discovery(self):
                 u"vsphere_host:host3", u"vsphere_type:vm"
             ]
         )
+
+
+class TestVsphereTopo(AgentCheckTest):
+
+    CHECK_NAME = "vsphere"
+
+    def vm_mock_content(self):
+        datastore = MockedMOR(_moId="54183927-04f91918-a72a-6805ca147c55")
+        hardware = MockedMOR(numCPU=1, memoryMB=4096)
+        config = MockedMOR(guestId='ubuntu64Guest', guestFullName='Ubuntu Linux (64-bit)', hardware=hardware)
+        view_MOR = MockedMOR(spec="VirtualMachine", name="Ubuntu",
+                             datastore=[datastore], config=config)
+
+        view_mock = MagicMock(view=[view_MOR])
+        viewmanager_mock = MagicMock(**{'CreateContainerView.return_value': view_mock})
+        content_mock = MagicMock(viewManager=viewmanager_mock)
+        return content_mock
+
+    def test_vsphere_objs(self):
```

Comment:
I would also like a test where we check the `regexes` parameter

Reply:
test (test_vsphere_vms_with_regex)  added for regex as well.

Reply:
👍